doc/newsfragments/2540_new.skip_remaining.rst

```diff
@@ -0,0 +1,3 @@
+Added a new command line argument ``--skip-remaining``, a new argument ``skip_strategy`` to MultiTest, allowing certain test execution being skipped for a better user experience.
```

Comment:
allow early return of test execution and skip remaining Testcases/Testsuites/Multitests being executed when a Testcase has failed or raised exception.